src/tiledb/cloud/bioimg/exportation.py

```diff
@@ -85,7 +85,7 @@ def export_tiff_udf(
             work,
             config,
             *args,
-            name=f"{task_prefix} - {i}/{num_batches}",
+            name=f"{task_prefix} - {i}",
```

Comment:
are these changes related?

Reply:
Nope as noted in PR description I just thought worth changing it since it is a misc bug.